public/playersBetController.js

```diff
@@ -0,0 +1,157 @@
+  // if an insurance bet is made, then the player starts his hand normally, that is, with the same options as always
+
+  controlsHtml() {
+    return `
+    <div class='bet-controls'>
+        <div class='bet-controls__pot'>Pot: ${this.activePlayer.pot}</div>
+        <div class='bet-controls__bet'>Current bet: ${this.currentBet}</div>    
+        <button type='button' class= 'bet-controls__btn btn--25'>
+        25
+        </button>
+        <button type='button' class= 'bet-controls__btn btn--50'>
+        50
+        </button>
+        <button type='button' class= 'bet-controls__btn btn--100'>
+        100
+        </button>    
+        <button type='button' disabled class= 'bet-controls__btn btn--clear'>
+        Clear
+        </button>
+        <button type='button' disabled class= 'bet-controls__btn btn--bet'>
+        Confirm bet
+        </button>
+    </div>    
+      `;
+  }
+
+  renderBetControls() {
+    this.controlsContainerEle.insertAdjacentHTML(
+      "beforeend",
+      this.controlsHtml()
+    );
+  }
+
+  initBetController() {
+    this.initPlayers();
+    this.renderBetControls();
+    this.selectDomElements();
+    this.addEventListeners();
+  }
+
+  initPlayers() {
+    this.players = this.board.players;
+    this.updateActivePlayer();
+  }
+
+  selectDomElements() {
+    this.betControls = document.querySelector(".bet-controls");
+    this.bet25 = this.controlsContainerEle.querySelector(".btn--25");
+    this.bet50 = this.controlsContainerEle.querySelector(".btn--50");
+    this.bet100 = this.controlsContainerEle.querySelector(".btn--100");
+    this.clear = this.controlsContainerEle.querySelector(".btn--clear");
+    this.currentPotCont =
+      this.controlsContainerEle.querySelector(".bet-controls__pot");
+    this.curretBetCont =
+      this.controlsContainerEle.querySelector(".bet-controls__bet");
+    this.confirm = this.controlsContainerEle.querySelector(".btn--bet");
+  }
+
+  addEventListeners() {
+    this.bet25.addEventListener("click", this.betting.bind(this));
+    this.bet50.addEventListener("click", this.betting.bind(this));
+    this.bet100.addEventListener("click", this.betting.bind(this));
+    this.clear.addEventListener("click", this.clearBet.bind(this));
+    this.confirm.addEventListener("click", this.confirmBet.bind(this));
+  }
+
+  updateActivePlayer() {
+    this.activePlayer = this.players[this.currentPlayerTurn];
+    document
+      .querySelector(`#${this.activePlayer.name} .card`)
+      .classList.add("active");
+  }
+
+  isLastPlayerTurn() {
+    const lastPlayerPosition = this.players.length - 2;
+    return lastPlayerPosition === this.currentPlayerTurn;
+  }
+
+  nextPlayer() {
+    document
+      .querySelector(`#${this.activePlayer.name} .card`)
+      .classList.remove("active");
```

Comment:
Same as previous comment, you can replace this with the active player `removeFocus` method